e2e/pages/caseView.page.js

```diff
@@ -34,7 +34,7 @@ module.exports = {
   async goToNewActions(actionSelected) {
     const currentUrl = await I.grabCurrentUrl();
     await I.retryUntilExists(async () => {
-      if(await I.waitForSelector(this.actionsDropdown, 10) != null) {
+      if(await I.waitForSelector(this.actionsDropdown, 60) != null) {
```

Comment:
give it a little more time to load entire page. Page refresh on aat is really slow (it loads a lot of js files plus ccd call, 10 sec is not enough)